pkg/injector/monitoring/metrics.go

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package monitoring
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	appID   = "app_id"
+	podName = "pod_name"
+)
+
+var (
+	successfulAdmissionReviewResponseTotal = stats.Int64(
+		"injector/successful_admissionreview_response_total",
```

Comment:
I would follow this metric name format: [component]/[subcomponent]/[operation]_total
more readable. 

e.g. injector/sidecar_injection_succeed_total